libs/datamodel/core/src/ast/mod.rs

```diff
@@ -104,6 +104,13 @@ impl SchemaAst {
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub(crate) struct ModelId(u32);
 
+impl ModelId {
+    /// Used for range bounds when iterating over BTrees.
+    pub(crate) const ZERO: ModelId = ModelId(0);
+    /// Used for range bounds when iterating over BTrees.
+    pub(crate) const MAX: ModelId = ModelId(u32::MAX);
```

Comment:
that's a lot of models ;-)

Reply:
We'll have to make sure to make that a `u64` when we implement multi-file schemas. You aren't enterprise until you have at least 2,147,483,647.models.